### PR TITLE
Highlight squares when hovering over them

### PR DESCRIPTION
**Pull request by thearst3rd**
For cards where all tiles do the same thing, highlight all of those tiles.

### CODE REVIEW

File: src/cards/actions/increase_health.gd [hunk 1]
```diff
@@ -33,3 +33,7 @@ func positive_effects(_grid_pos: Vector2i) -> Array[Vector2i]:
 		Vector2i(8, 4), Vector2i(9, 4), Vector2i(10, 4),
 		Vector2i(8, 5), Vector2i(9, 5), Vector2i(10, 5),
 	]
+
+
+func hovering_tiles(_grid_pos: Vector2i) -> Array[Vector2i]:
+	return positive_effects(_grid_pos)
```
Comment (admoore0) [inline src/cards/actions/increase_health.gd:39]: No underscore

File: src/cards/actions/oil.gd [hunk 2]
```diff
@@ -36,3 +36,7 @@ func perform_action(grid_pos: Vector2i, is_enemy: bool) -> void:
 
 func negative_effects(_grid_pos: Vector2i) -> Array[Vector2i]:
 	return [Vector2i(7, 3), Vector2i(7, 4), Vector2i(7, 5)]
+
+
+func hovering_tiles(_grid_pos: Vector2i) -> Array[Vector2i]:
+	return negative_effects(_grid_pos)
```
Comment (admoore0) [inline src/cards/actions/oil.gd:42]: No underscore

File: src/states/game/game.tscn [hunk 3]
```diff
@@ -154,6 +157,7 @@ offset_bottom = 177.0
 size_flags_horizontal = 0
 theme_override_styles/panel = SubResource("StyleBoxFlat_biu44")
 metadata/_edit_group_ = true
+metadata/_edit_lock_ = true
```
Comment (admoore0) [inline src/states/game/game.tscn:160]: based editor locking

**Review by admoore0 — CHANGES_REQUESTED** [review]

File: src/cards/actions/increase_health.gd (hunk 1 above)
Comment (thearst3rd) [reply src/cards/actions/increase_health.gd:39]: yes underscore :)

**Review by kdreese — DISMISSED** [review]

**Review by admoore0 — APPROVED** [review]

**Review by kdreese — APPROVED** [review]

**Pull request merged**